plottable.d.ts

```diff
@@ -3056,6 +3056,32 @@ declare module Plottable {
              * @constructor
              */
             constructor();
+            /**
+             * Gets the interpolation function associated with the plot.
+             *
+             * @return {string | (points: Array<[number, number]>) => string)}
+             */
+            interpolator(): string | ((points: Array<[number, number]>) => string);
+            interpolator(interpolator: "linear"): Line<X>;
```

Comment:
These signatures should be grouped with the other setter signature, under the TSDoc block for the setter, right?